src/components/standalone/firewall/rules/FirewallRulesTable.vue

```diff
@@ -97,6 +97,11 @@ const filteredRules = computed(() => {
   }
 })
 
+// enable follows rule.enabled and rule.active_zone
+const enabled = computed(() => {
```

Comment:
`computed` values usually don't have parameters (`rule` in this case). Computed values with parameters are not [officially documented](https://vuejs.org/guide/essentials/computed.html#computed-properties) and probably don't give any advantages. I suggest to use a simple `isEnabled` method instead